### PR TITLE
Add command to turn off swap in install-kubeadm.md

### PR DESCRIPTION
**Pull request by lichao127**
add swapoff command

### CODE REVIEW

**Review by T-Lakshmi — COMMENTED** [review]
Hey @lichao127,
Thanks you for your PR,
As per the community standards, it is required to sign the CLA before the PR can be reviewed,
please sign CLA. You can follow the steps documented here: [CLA Steps](https://git.k8s.io/community/CLA.md)

**Review by neolit123 — COMMENTED** [review]
we used to have this command here, but then it was removed. IIRC swapoff was not a portable Linux command. i don't remember exactly.

**Comment by lichao127** [comment]
> [T-Lakshmi](/T-Lakshmi)

Hi @T-Lakshmi I have signed the CLA as individual contributor.

**Comment by lichao127** [comment]
> 

Was it because an additional change to `/etc/fstab` is needed? 

Found an Ansible example in https://github.com/kubernetes/website/blob/main/content/en/blog/_posts/2019-03-15-Kubernetes-setup-using-Ansible-and-Vagrant.md#step-22-kubelet-will-not-start-if-the-system-has-swap-enabled-so-we-are-disabling-swap-using-the-below-code

File: content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ see the [Creating a cluster with kubeadm](/docs/setup/production-environment/too
 * Unique hostname, MAC address, and product_uuid for every node. See [here](#verify-mac-address) for more details.
 * Certain ports are open on your machines. See [here](#check-required-ports) for more details.
 * Swap disabled. You **MUST** disable swap in order for the kubelet to work properly.
+    * `sudo swapoff -a`
```
Comment (sftim) [inline content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:30]:
```suggestion
    * For example: `sudo swapoff -a` (and make sure to make the change permanent)
``` 
There is no one consistent way to disable swap space on nodes. For example, you can also define a `.swap` unit in systemd, or have an EFI partition of type `0657fd6d-a4ab-43c4-84e5-0933c84b4f4f` that autodiscovers. The suggestion here would not make permanent changes which is likely to surprise readers.

Comment (lichao127) [reply content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md]: updated in https://github.com/kubernetes/website/pull/40654/commits/88cb81f65ea34125c4f5b127d07cf12f622e7fdd

File: content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 2]
```diff
@@ -27,6 +27,7 @@ see the [Creating a cluster with kubeadm](/docs/setup/production-environment/too
 * Unique hostname, MAC address, and product_uuid for every node. See [here](#verify-mac-address) for more details.
 * Certain ports are open on your machines. See [here](#check-required-ports) for more details.
 * Swap disabled. You **MUST** disable swap in order for the kubelet to work properly.
+    * For example: `sudo swapoff -a` (and make sure this change is permanent. Swap may be set in files like `/etc/fstab`, `systemd.swap` depending how it was configured on your system)
```
Comment (tengqm) [inline content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:30]:
```suggestion
  For example, on some Linux systems, you can do `sudo swapoff -a`.
  To make this change persistent across reboots, wwap may be set in files like
  `/etc/fstab`, `systemd.swap` depending how it is configured on your system.
```

Comment (lichao127) [reply content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md]: Updated in https://github.com/kubernetes/website/pull/40654/commits/b2b8299841f45e5da35fa474a019906fdda056fd

**Comment by tengqm** [comment]
please squash your pr into a single commit

**Comment by lichao127** [comment]
> please squash your pr into a single commit

commits have been squashed

**Pull request merged**